refract-spec.md

```diff
@@ -54,7 +50,7 @@ The Refract Element contains three properties, `element`, `attributes`, and `con
 
 #### Example
 
-An simple element may look like this, where `foo` is the element type, `id` is an attribute for the `foo` element, and `content` is a string with a value of `bar`.
+An element may look like this, where `foo` is the element type, `id` is an attribute for the `foo` element, and `content` is a string with a value of `bar`.
```

Comment:
I think you should change this given that id='4' is probably not compliant with and id element that probably should be a string. Not sure, but at the very least, not sure this is a good example anymore.